sp800-63b/references.md

```diff
@@ -63,52 +63,52 @@
+<a name="SP800-63C"></a>[SP 800-63C] NIST Special Publication 800-63C, *Digtial Identity Guidelines: Assertions and Federation*, June 2017, <https://doi.org/10.6028/NIST.SP.800-63c>.
 
-<a name="SP800-90Ar1></a>[SP 800-90Ar1] NIST Special Publication 800-90A Revision 1, *Recommendation for Random Number Generation Using Deterministic Random Bit Generators*, June 2015, available at: <http://dx.doi.org/10.6028/NIST.SP.800-90Ar1>.
+<a name="SP800-90Ar1"></a>[SP 800-90Ar1] NIST Special Publication 800-90A Revision 1, *Recommendation for Random Number Generation Using Deterministic Random Bit Generators*, June 2015, <http://dx.doi.org/10.6028/NIST.SP.800-90Ar1>.
 
-<a name="SP800-107"></a>[SP 800-107] NIST Special Publication 800-107 Revision 1, *Recommendation for Applications Using Approved Hash Algorithms*, August 2012, available at: <http://dx.doi.org/10.6028/NIST.SP.800-107r1>.
+<a name="SP800-107"></a>[SP 800-107] NIST Special Publication 800-107 Revision 1, *Recommendation for Applications Using Approved Hash Algorithms*, August 2012, <http://dx.doi.org/10.6028/NIST.SP.800-107r1>.
 
-<a name="SP800-131A"></a>[SP 800-131A] NIST Special Publication 800-131A Revision 1, *Transitions: Recommendation for Transitioning the Use of Cryptographic Algorithms and Key Lengths*, November 2015, available at: <http://dx.doi.org/10.6028/NIST.SP.800-131Ar1>
+<a name="SP800-131A"></a>[SP 800-131A] NIST Special Publication 800-131A Revision 1, *Transitions: Recommendation for Transitioning the Use of Cryptographic Algorithms and Key Lengths*, November 2015, <http://dx.doi.org/10.6028/NIST.SP.800-131Ar1>
 
-<a name="SP800-132"></a>[SP 800-132] NIST Special Publication 800-132, *Recommendation for Password-Based Key Derivation*, December 2010, available at: <http://dx.doi.org/10.6028/NIST.SP.800-132>.
+<a name="SP800-132"></a>[SP 800-132] NIST Special Publication 800-132, *Recommendation for Password-Based Key Derivation*, December 2010, <http://dx.doi.org/10.6028/NIST.SP.800-132>.
 
-<a name="SP800-185"></a>[SP 800-185] NIST Special Publication 800-185, *SHA-3 Derived Functions: cSHAKE, KMAC, TupleHash, and ParallelHash*, December, 2016, available at: <https://doi.org/10.6028/NIST.SP.800-185>.
+<a name="SP800-185"></a>[SP 800-185] NIST Special Publication 800-185, *SHA-3 Derived Functions: cSHAKE, KMAC, TupleHash, and ParallelHash*, December, 2016, <https://doi.org/10.6028/NIST.SP.800-185>.
 
 ### 4. Federal Information Processing Standard
 
-<a name="FIPS140-2"></a>[FIPS 140-2] Federal Information Processing Standard Publication 140-2, *Security Requirements for Cryptographic Modules*, May 25, 2001 (with Change Notices through December 3, 2002), available at: <https://doi.org/10.6028/NIST.FIPS.140-2>.
+<a name="FIPS140-2"></a>[FIPS 140-2] Federal Information Processing Standard Publication 140-2, *Security Requirements for Cryptographic Modules*, January 11, 1994 (with Change Notices through December 3, 2002), <https://doi.org/10.6028/NIST.FIPS.140-2>.
```

Comment:
January 11, 1994 was the effective date of FIPS 140-1. FIPS 140-2 is dated May 25, 2001.